services/grip/frontend/.env.test

```diff
@@ -0,0 +1 @@
+VITE_TRACTION_ENDPOINT=https://traction-api-test.apps.silver.devops.gov.bc.ca:5100
```

Comment:
We'll have to adjust how the environmental settings for Traction URLs get into the app at deployment time when we figure out our hosting and Helm/OCP plans
https://apps.nrs.gov.bc.ca/int/jira/browse/TRACTION-314

Reply:
Yeah. I'm starting to think we should either configure NGinx to proxy the API or maybe spin up a little Node server to serve the assets and proxy.